### PR TITLE
Remove all issue templates from repo in favor of developer portal Support page

### PR DESCRIPTION
**Pull request by mjlumetta**
Removes issue templates from this repo per [API-3990](https://vajira.max.gov/browse/API-3990).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ If you are a third party seeking access to VA Lighthouse APIs, you will need to
 
 ## Further Assistance
 
-If you have any questions or need further assistance, please visit our [support page](https://developer.va.gov/support), where you can contact us and peruse the FAQs. You will receive a human reply within 48 hours. Additionally, you may open a comment or support request [inside this repo](https://github.com/department-of-veterans-affairs/vets-api-clients/issues/new/choose).
+If you have any questions or need further assistance, please visit our [support page](https://developer.va.gov/support), where you can contact us and peruse the FAQs. You will receive a human reply within 48 hours.
```
Comment (gileswells) [inline README.md:64]: There's a mismatch on the estimated time for response. On the dev portal it lists one business day as the expectation.

**Review by gileswells — CHANGES_REQUESTED** [review]
I would like to see a new config.yml file added to `.github/ISSUE_TEMPLATE/` to direct people to the developer portal's support pages.

https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

File: README.md (hunk 1 above)
Comment (mjlumetta) [reply README.md:64]: @ryanlink thoughts on this?

Comment (ryanlink) [reply README.md:64]: LGTM!

Comment (ryanlink) [reply README.md:64]: hm.  will take this up with marmoset, hang tight

Comment (ryanlink) [reply README.md:64]: Make it 24 hours to be consistent with the contact us page, please.

Comment (mjlumetta) [reply README.md]: updated! thanks for clarifying, Ryan, and thanks for catching it Giles

**Review by gileswells — APPROVED** [review]
LGTM

**Pull request merged**